code/__HELPERS/announcements.dm

```diff
@@ -74,5 +73,12 @@
 /proc/create_announcement_div(message, color = "default")
 	return "<div class='chat_alert_[color]'>[message]</div>"
 
+/**
+ * Inserts a span styled message into an OOC alert style div
+ *
+ *
+ * Arguments
+ * * message - required, the message contents
+ */
 /proc/create_ooc_announcement_div(message)
 	return "<div class='ooc_alert'>[message]</div>"
```

Comment:
why aren't these macros? it doesn't matter since the proc overhead is irrelevant here but just curious

Reply:
suppose they could. I thought about starting a div.dm, but for just two wasn't sure if it was worth it.